<launch>
    <node pkg="turtlesim" type="turtlesim_node" name="sim"/>
    <node pkg="turtlesim" type="turtle_teleop_key" name="teleop" output="screen"/>

    <node name="turtle1_tf_broadcaster" pkg="hello_tutorials"
        type="turtle_tf_broadcaster.py" respawn="false" output="screen" >
        <param name="turtle" type="string" value="turtle1" />
    </node>

    <node name="turtle2_tf_broadcaster" pkg="hello_tutorials"
        type="turtle_tf_broadcaster.py" respawn="false" output="screen" >
        <param name="turtle" type="string" value="turtle2" />
    </node>

    <node name="listener" pkg="hello_tutorials" type="turtle_tf_travel_listener.py"
        output="screen"/>
</launch>